FluentScreenRecorder.sln

```diff
@@ -67,6 +69,26 @@ Global
 		{4BA5485A-9122-404C-8FE3-15A8A0E31E40}.Release|x86.ActiveCfg = Release|x86
 		{4BA5485A-9122-404C-8FE3-15A8A0E31E40}.Release|x86.Build.0 = Release|x86
 		{4BA5485A-9122-404C-8FE3-15A8A0E31E40}.Release|x86.Deploy.0 = Release|x86
+		{123D7523-1B05-4C05-A2FA-6A14314A16F3}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{123D7523-1B05-4C05-A2FA-6A14314A16F3}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{123D7523-1B05-4C05-A2FA-6A14314A16F3}.Debug|ARM.ActiveCfg = Debug|ARM
+		{123D7523-1B05-4C05-A2FA-6A14314A16F3}.Debug|ARM.Build.0 = Debug|ARM
+		{123D7523-1B05-4C05-A2FA-6A14314A16F3}.Debug|ARM64.ActiveCfg = Debug|ARM64
+		{123D7523-1B05-4C05-A2FA-6A14314A16F3}.Debug|ARM64.Build.0 = Debug|ARM64
+		{123D7523-1B05-4C05-A2FA-6A14314A16F3}.Debug|x64.ActiveCfg = Debug|x64
+		{123D7523-1B05-4C05-A2FA-6A14314A16F3}.Debug|x64.Build.0 = Debug|x64
+		{123D7523-1B05-4C05-A2FA-6A14314A16F3}.Debug|x86.ActiveCfg = Debug|x86
+		{123D7523-1B05-4C05-A2FA-6A14314A16F3}.Debug|x86.Build.0 = Debug|x86
+		{123D7523-1B05-4C05-A2FA-6A14314A16F3}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{123D7523-1B05-4C05-A2FA-6A14314A16F3}.Release|Any CPU.Build.0 = Release|Any CPU
+		{123D7523-1B05-4C05-A2FA-6A14314A16F3}.Release|ARM.ActiveCfg = Release|ARM
+		{123D7523-1B05-4C05-A2FA-6A14314A16F3}.Release|ARM.Build.0 = Release|ARM
+		{123D7523-1B05-4C05-A2FA-6A14314A16F3}.Release|ARM64.ActiveCfg = Release|ARM64
+		{123D7523-1B05-4C05-A2FA-6A14314A16F3}.Release|ARM64.Build.0 = Release|ARM64
+		{123D7523-1B05-4C05-A2FA-6A14314A16F3}.Release|x64.ActiveCfg = Release|x64
+		{123D7523-1B05-4C05-A2FA-6A14314A16F3}.Release|x64.Build.0 = Release|x64
+		{123D7523-1B05-4C05-A2FA-6A14314A16F3}.Release|x86.ActiveCfg = Release|x86
+		{123D7523-1B05-4C05-A2FA-6A14314A16F3}.Release|x86.Build.0 = Release|x86
```

Comment:
Changes to the build architecture need to be reverted, I don't want anycpu/arm configuration and I don't understand the other changes.